src/main.cpp

```diff
@@ -125,7 +172,9 @@ int main(const int argc, const char *const *const argv) {
         return false;
     };
 
-    std::copy_if(QtModules.begin(), QtModules.end(), std::back_inserter(foundQtModules),
+    const std::vector<QtModule>& qtmodules = getQtModules(qtMajorVersion);
```

Comment:
Code style: We use camelCase, so this should be `qtModules`.

Reply:
Ok, applied.